src/client/common/errors/errors.ts

```diff
@@ -38,6 +45,13 @@ function tagWithWin32Error(stdErr: string, tags: string[] = []) {
         tags.push('win32api');
     }
 }
+function tagWithChildProcessExited(stdErrOrStackTrace: string, tags: string[] = []) {
```

Comment:
Found a few `unknown` errors & looking at the stack trace I can tell we're getting a few of these `ChildProcess.exithandler`, figured i might as well classify them.
Seems to be a kernel has died, but not yet classifying them as kernel died, as we're just getting a generic node error, however the subcategory/tag will help classify these.